### PR TITLE
Adding Requirement to Update Release Notes to the PR checklist

### PR DESCRIPTION
**Pull request by MaxwellKendall**
### Check List Before Merging! (For the Author)
- [ ] Story Book has been rebuilt (new build artifacts in `/docs`)
- [ ] Components have been rebuilt (new build artifacts in `/dist`)
- [ ] All new components have been exported in `/index.js`

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
+Thank you for taking the time to contribute to this library.
```
Comment (jonhill13) [inline pull_request_template.md]: Love it

File: draft_release_notes.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+**Description of Previous Approach**
+
+**Description of New Approach**
+
+**Benefits of New Approach**
+
+**Integration**
+[USAS/Broker PR here](https://github.com/fedspendingtransparency/usaspending-website/pull/1431):
+
+- [] Development Complete
```
Comment (jonhill13) [inline draft_release_notes.md]: Wouldn't development already have been complete upon PR creation?

File: draft_release_notes.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- [] Design Complete
```
Comment (jonhill13) [inline draft_release_notes.md]: Design Review Complete

File: draft_release_notes.md (hunk 2 above)
Comment (jonhill13) [reply draft_release_notes.md]: What is the reasoning behind having tasks in the release notes, shouldn't these be moved or are already included in the PR checklist.

Comment (MaxwellKendall) [reply draft_release_notes.md]: B/c we might reasonably expect the release notes to be created before the development is done.

File: draft_release_notes.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+```markdown
+## <Feature> added to DT-UI
+**Description of Previous Approach**
```
Comment (jonhill13) [inline draft_release_notes.md]: Is this a standard when using a component library Previous Approach, New Approach, Benefits?

I thought release notes followed a pattern like this:
`# Upgrade Steps
- [ACTION REQUIRED]
-

# Breaking Changes
- 
- 

# New Features
-
-

# Bug Fixes
-
-

# Improvements
-
-

# Other Changes
-
-`

File: draft_release_notes.md (hunk 2 above)
Comment (jonhill13) [reply draft_release_notes.md]: I do not believe release notes should have a checklist. Release notes should be informing users of changes.  We could add a check list for release notes PRs.

File: pull_request_template.md [hunk 5]
```diff
@@ -1,4 +1,9 @@
...
+
 ### Check List Before Merging! (For the Author)
 - [ ] Story Book has been rebuilt (new build artifacts in `/docs`)
 - [ ] Components have been rebuilt (new build artifacts in `/dist`)
 - [ ] All new components have been exported in `/index.js`
+- [ ] Draft release notes have been updated describing these changes ([use this template](https://github.com/fedspendingtransparency/data-transparency-ui/blob/master/draft_release_notes.md) to document your changes)
+- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated with new component and all instances of that component in USASpending. Refer to inventory documentation [here](https://airtable.com/tblHeQOwctk5z5FOj/viw2T0ewpuDPOhHtL?blocks=show) for help with identifying instances of components. `if applicable`
+
```
Comment (jonhill13) [inline pull_request_template.md]: Should we add checkbox for broker similar to the Component Library Integration Status Report?

Comment (MaxwellKendall) [reply pull_request_template.md]: I think the integration report should account for both broker and USAS.

File: draft_release_notes.md (hunk 4 above)
Comment (MaxwellKendall) [reply draft_release_notes.md]: This looks better to me...

File: draft_release_notes.md (hunk 2 above)
Comment (MaxwellKendall) [reply draft_release_notes.md]: Understood!!

**Pull request merged**